Modules/AuctionManager/GUI.lua

```diff
@@ -272,7 +275,7 @@ function AuctionManagerGUI:GenerateAuctionOptions()
         award_value = {
             name = "Award value",
             type = "input",
-            set = (function(i,v) self.awardValue = tonumber(v) or 0 end),
+            set = (function(i,v) AuctionManagerGUI:setInputAwardValue(self, v) end),
```

Comment:
Syntactic sugar: `Array:Method()` is same as `Array.Method(self)` thus we don't pass `self` when using `:`